src/components/ListCard.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+// TODO find something better
+const DEFAULT_IMAGE_URL = "https://upload.wikimedia.org/wikipedia/commons/thumb/7/79/2010-brown-bear.jpg/200px-2010-brown-bear.jpg";
+
+const styles = theme => ({
+  picture: {
+    width: 150,
```

Comment:
Do we need both min and regular height and widths here?

Reply:
Nope!